news/_posts/2020-03-bug-bounty-program.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title:  "Let the hunt begin"
```

Comment:
A more explicit title would be great, including the keywork "bug bounty" for SEO purpose.

Reply:
SEO doesn't use only the title, it's an old usage, it'll take all content information on the page, sometimes the meta description, the content, and also keywords are better than the title. But anyway, I'm waiting for someone else feedback

Reply:
Let the hunt begin would be a perfect subtitle, btw

Reply:
:cry: